src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -948,6 +945,18 @@ private boolean shouldAvoidGeneratingExterns(String filePath, String symbolName)
     return false;
   }
 
+  /**
+   * Strip window. or this. from full symbol names. Technically, window. and this. symbols are
+   * tracked separately in the type system from the global symbols, but they largely overlap both in
+   * TS and Closure.
+   */
+  private String normalizeWindowGlobals(String name) {
+    if (name.startsWith("window.") || name.startsWith("this.")) {
```

Comment:
The 'if' check isn't needed here, right?  replaceAll will only match if these are true?

Reply:
right, of course. Bad refactoring.